BitmarkRegistry/Services/KeychainStore.swift

```diff
@@ -31,11 +33,29 @@ class KeychainStore {
     return getDataFromKeychain(key: bitmarkSeedCoreKey)
   }
 
-  private static func getDataFromKeychain(key: String) -> Data? {
+  // *** jwt ***
+  static func saveToKeychain(jwt: String) throws {
```

Comment:
This JWT is designed to be short-lived. We can just keep it in memory, not need to be in keychain. Keychain is also a good option, but not needed 😛.

Reply:
ok, thanks for your feedback. I will update it :)